File: AutoDoc/src/main/CodeDAO.java
import java.sql.PreparedStatement;
import javax.swing.JOptionPane;
import java.sql.Connection;

public class CodeDAO {
	
	Connection conn;
	PreparedStatement pstm;
	
	public void sendCodeDatabase(CodeDTO objCodeDTO) {
		
		String sql = "INSERT codes(code, cpf) VALUES (?,?)";
		
		conn = new ConnectionDAO().connectJDBC();
		
		try {
			pstm = conn.prepareStatement(sql);
			pstm.setString(1, objCodeDTO.getCode());
			pstm.setString(2, objCodeDTO.getCpf());
			
			pstm.execute();
			pstm.close();
		} 
		catch (Exception error) {
			JOptionPane.showMessageDialog(null, "CodeDAO: "+error);
		}
	}
	
}

File: medtec/connection/CodeDAO.php
<?php

class CodeDAO {

    private $connection;

    public function __construct() {
        $this -> connection = new Connection();
    }

    public function validate($code, $cpf) {
        $sql = "SELECT * FROM codes WHERE code = '$code' AND cpf = '$cpf'";

        $execute = mysqli_query($this -> connection -> getConn(), $sql);
        
        if (mysqli_num_rows($execute) > 0) {
            return true;
        }
        else {
            return false;
        }
    }
}